docs/Quantization.md

```diff
@@ -102,6 +102,17 @@ the graph.
 ./bin/image-classifier tests/images/imagenet/*.png -image_mode=0to1 -m=resnet50 -load_profile="profile.yaml"
 ```
 
+By default, all nodes that can be quantized will be quantized. However, we may
+only want to quantize some parts of a model, e.g. if accuracy loss is too high
+when all node kinds are quantized. The Glow loader currently allows for
+disabling quantization of all nodes of a specific kind which are found in the
+graph. For example, if the loaded model sees high accuracy loss when
+element-wise Add is quantized, it can be left in floating point. This can be
+done by passing on the command line the node name via the option
+`-do_not_quantize_node`. For example, to not quantize the Add node when running
+the quantized text translator: ``` ./bin/text-translator -m en2gr
+-load_profile=en2gr.yaml -do_not_quantize_node=Add ```
```

Comment:
can you mention that `do_not_quantize_node` could contain several node types? I got a sense that it's only a single node type can be blacklisted here.

Reply:
Is there a more complete way to do that?
Such as an sub-group of nodes be disabled for quantize?

Reply:
Sure -- just to be clear I believe the way to add multiple nodes here is to specify each one individually, e.g. `-do_not_quantize_node=Add -do_not_quantize_node=Div` etc. Hence why I made it node and not nodes. Is that not the case, is there a way to do all under one flag?

Reply:
@lygstate Are you talking about specifying multiple Node kinds, as I replied above? Or by a sub-group of nodes do you mean that only some of the nodes with the Node kind would be quantized?

Reply:
Using comma should work as well, IIRC the LLVM option parsing:
 `-do_not_quantize_node=Add,Div`

Reply:
Figured it out -- I needed to add `llvm::cl::CommaSeparated` to enable that. Thanks @qcolombet!